message.go

```diff
@@ -62,12 +64,20 @@ func (c *TunaSessionClient) getOrComputeSharedKey(remotePublicKey []byte) (*[sha
 	box.Precompute(sharedKey, curve25519PublicKey, curveSecretKey)
 
 	c.Lock()
-	c.sharedKeys[string(remotePublicKey)] = sharedKey
+	c.sharedKeys[k] = sharedKey
 	c.Unlock()
 
 	return sharedKey, nil
 }
 
+func (c *TunaSessionClient) GetAccountPubKey() []byte {
+	return c.clientAccount.PubKey()
+}
+
+func (c *TunaSessionClient) GetAccountPrivKey() []byte {
```

Comment:
Don't add this API unless absolutely needed. It could be dangerous because it exposes private key